File: coreui/src/main/java/piuk/blockchain/androidcoreui/utils/AndroidUtils.java
package piuk.blockchain.androidcoreui.utils;

import android.os.Build;

public class AndroidUtils {

    public static boolean is25orHigher() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1;
    }

    public static boolean is26orHigher() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O;
    }

}

File: app/src/main/java/piuk/blockchain/android/util/AndroidUtils.kt
package piuk.blockchain.android.util

import android.os.Build

object AndroidUtils {
    fun is25orHigher(): Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1
    fun is26orHigher(): Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
}